app/build.gradle.kts

```diff
@@ -0,0 +1,336 @@
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+    val qaImplementation by configurations
+
+    implementation(fileTree("libs") { include("*.jar") })
+
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Dependencies.kotlin}")
+
+    implementation("com.crashlytics.sdk.android:crashlytics:2.9.8")
+
+    // Support libraries
+    val supportAnnotationLib = "androidx.annotation:annotation:1.0.1"
+    implementation(supportAnnotationLib)
+    implementation("androidx.appcompat:appcompat:1.0.2")
+    implementation("androidx.recyclerview:recyclerview:1.0.0")
+    implementation("com.google.android.material:material:1.0.0")
+
+    // architecture components
+    val lifecycleVersion = "2.0.0"
+    implementation("androidx.lifecycle:lifecycle-extensions:$lifecycleVersion")
+    kapt("androidx.lifecycle:lifecycle-compiler:$lifecycleVersion")
+    implementation("androidx.lifecycle:lifecycle-reactivestreams:$lifecycleVersion")
+
+    // Google play services
+    val googlePlayVersion = "16.1.0"
+    implementation("com.google.android.gms:play-services-base:$googlePlayVersion")
+
+    // Rx
+    implementation("io.reactivex.rxjava2:rxjava:2.2.5")
+    implementation("io.reactivex.rxjava2:rxandroid:2.1.0")
+    implementation("io.reactivex.rxjava2:rxkotlin:2.3.0")
+    implementation("com.jakewharton.rxrelay2:rxrelay:2.1.0")
+    implementation("com.jakewharton.rxbinding2:rxbinding-kotlin:2.2.0")
+
+    // Jake Wharton/Square
+    val retrofitVersion = "2.5.0"
+    implementation("com.squareup.retrofit2:retrofit:$retrofitVersion")
+    implementation("com.squareup.retrofit2:converter-gson:$retrofitVersion")
+    implementation("com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion")
+    implementation("com.google.code.gson:gson:2.8.5")
+
+    val okhttpVersion = "3.12.0"
+    implementation("com.squareup.okhttp3:okhttp:$okhttpVersion")
+    implementation("com.squareup.okhttp3:logging-interceptor:$okhttpVersion")
+
+    val butterKnifeVersion = "10.0.0"
+    implementation("com.jakewharton:butterknife:$butterKnifeVersion")
+    kapt("com.jakewharton:butterknife-compiler:$butterKnifeVersion")
+    implementation("com.jakewharton.timber:timber:4.7.1")
+    implementation("com.squareup.picasso:picasso:2.71828")
+
+    // Other
+    implementation("io.intrepid.commonutils:commonutils:0.2.3")
+
+    // LeakCanary
+    val leakCanaryVersion = "1.6.3"
+    debugImplementation("com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion")
+    releaseImplementation("com.squareup.leakcanary:leakcanary-android-no-op:$leakCanaryVersion")
+    qaImplementation("com.squareup.leakcanary:leakcanary-android-no-op:$leakCanaryVersion")
+    testImplementation("com.squareup.leakcanary:leakcanary-android-no-op:$leakCanaryVersion")
+
+    // Common test dependencies
+    val junitLib = "junit:junit:4.12"
+    val mockitoVersion = "2.23.4"
```

Comment:
Should we put version `val`s in Dependencies.kt?

Reply:
That would definitely be the case if it's used across multiple modules/ gradle files. Since it's currently only used in this block, there's not much reason to move it. I don't feel strongly about it either way though.

Reply:
I'm fine with that